x/nft/client/cli/query.go

```diff
@@ -64,9 +64,9 @@ func GetCmdQuerySupply() *cobra.Command {
 			}
 
 			// nolint: govet
-			if err := types.ValidateDenomID(args[0]); err != nil {
-				return err
-			}
+			// if err := types.ValidateDenomID(args[0]); err != nil {
+			// 	return err
+			// }
 
```

Comment:
why is ValidateDenomID commented out?

Reply:
Created a new function to validate that also checks for ibc denom ids.